source/utilities.tex

```diff
@@ -40,7 +40,7 @@
 \indextext{\idxhdr{utility}}%
 \indexlibrary{\idxhdr{utility}}%
 \indexlibrary{\idxcode{rel_ops}}%
-\synopsis{Header \tcode{<utility>} synopsis}
+\rSec2[utility.syn]{Header \tcode{<utility>} synopsis}
```

Comment:
Yeah, maybe -- I have never managed to make up my mind about this one. (And I have looked at this case before, and others awkward cases like `<algoritms>`.) Not sure what the best way is to present this.